python/doc/examples/meta_modeling/chaos_distribution_transformation.ipynb

```diff
@@ -0,0 +1,300 @@
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "T = ot.DistributionTransformation(X,Z)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We apply this transform on the sample generated from `X`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<TABLE><TR><TD></TD><TH>y0</TH></TR>\n",
+       "<TR><TH>0</TH><TD>0.3315264</TD></TR>\n",
+       "<TR><TH>1</TH><TD>1.189127</TD></TR>\n",
+       "<TR><TH>2</TH><TD>-1.101791</TD></TR>\n",
+       "<TR><TH>3</TH><TD>-1.84522</TD></TR>\n",
+       "<TR><TH>4</TH><TD>-0.3932781</TD></TR>\n",
+       "</TABLE>"
+      ],
+      "text/plain": [
+       "class=Sample name=Unnamed implementation=class=SampleImplementation name=Unnamed size=5 dimension=1 description=[y0] data=[[0.331526],[1.18913],[-1.10179],[-1.84522],[-0.393278]]"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "T(sampleX)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The inverse transform maps $Z$ into $X$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Tinverse = T.inverse()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<TABLE><TR><TD></TD><TH>y0</TH></TR>\n",
+       "<TR><TH>0</TH><TD>52358.82</TD></TR>\n",
+       "<TR><TH>1</TH><TD>27988.66</TD></TR>\n",
+       "<TR><TH>2</TH><TD>23916.19</TD></TR>\n",
+       "<TR><TH>3</TH><TD>26048.09</TD></TR>\n",
+       "<TR><TH>4</TH><TD>35798.44</TD></TR>\n",
+       "</TABLE>"
+      ],
+      "text/plain": [
+       "class=Sample name=Unnamed implementation=class=SampleImplementation name=Unnamed size=5 dimension=1 description=[y0] data=[[52358.8],[27988.7],[23916.2],[26048.1],[35798.4]]"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Tinverse(sampleZ)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Conclusion\n",
+    "\n",
+    "Most of the times, as a user of the library, we do not use the `DistributionTransformation` class directly, as this is done automatically by the polynomial chaos classes. However, it helps to understand how this transform is done, because this clarifies why the coefficients of the chaos expansion cannot be related to the input random variable $X$: the expansion is based on the standard variables $\\xi$. Hence, the absolute values of the corresponding coefficient has no straightforward interpretation even though squaring them allows to see the the part of variance associated with each coefficient. "
```

Comment:
```suggestion
    "The `DistributionTransformation` class is rarely used directly because the polynomial chaos classes perform the transformation automatically. However, understanding how this transform is done clarifies why the coefficients of the chaos expansion cannot be related to the input random variable $X$: the expansion is based on the standard variables $\\xi$. Hence, the absolute values of the corresponding coefficients have no straightforward interpretation, even though squaring them yields the part of the global variance associated with each coefficient. "
```